android-components/components/lib/crash/build.gradle

```diff
@@ -93,3 +93,59 @@ apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 apply from: '../../../android-lint.gradle'
 apply from: '../../../publish.gradle'
 ext.configurePublish(config.componentsGroupId, archivesBaseName, project.ext.description)
+
+/*
+ * Remote crash metrics logic.
+ *
+ * The below pulls metrics and pings from gecko-dev and ensures that they are processed by the glean
+ * parser.
+ */
+
+// The directory to store downloaded files.
+def gleanCrashFilesDir = new File(buildDir, "glean_crash")
+
+task downloadCrashMetrics(type: DownloadCrashGleanDefinitions) {
```

Comment:
Linking to the comment on the other PR: https://github.com/mozilla-mobile/firefox-android/pull/671/files#r1095076484

Reply:
Updated the comment above. I think we need to do something here e.g., https://github.com/mozilla-mobile/firefox-android/pull/671/files#r1095117634 which would make sure we always have the correct yaml files (from the matching Gecko version) on all branches (nightly/beta/release), instead of the hard-coded tag which can get out-of-sync easily.